lib/simulator-xcode-6.js

```diff
@@ -103,19 +154,23 @@ class SimulatorXcode6 {
     let files = [
       'Library/ConfigurationProfiles',
       'Library/Cookies',
-      'Library/DeviceRegistry.state',
       'Library/Logs',
       'Library/Preferences/.GlobalPreferences.plist',
-      'Library/Preferences/com.apple.Preferences.plist',
       'Library/Preferences/com.apple.springboard.plist',
       'var/run/syslog.pid'
     ];
+    if (await this.getPlatformVersion() !== '7.1') {
+      files.push('Library/DeviceRegistry.state');
+      files.push('Library/Preferences/com.apple.Preferences.plist');
+    } else {
+      files.push('Applications');
+    }
 
     files = files.map((s) => {
       return path.resolve(this.getDir(), s);
     });
 
-    let existence = files.map((f) => {
+    let existence = files.map(async (f) => {
```

Comment:
funny, i just added asyncMap to asyncbox, so instead you could do:

```js
let existence = await asyncMap(files.map, async (f) => { return util.hasAccess(f); });
```

it runs in parallel automatically.

up to you.